static/css/artist/artist_list.css

```diff
@@ -70,3 +73,35 @@
 .nav-float {
     float: left;
 }
+
+.artist-name {
+    padding-top: 5px;
+    padding-bottom: 0px;
+    font-family: Dosis, sans-serif;
+    background-color: #ecf0f1;
+}
+
+.artist-location {
+    padding-top: 5px;
+    padding-bottom: 10px;
+    font-family: Dosis, sans-serif;
```

Comment:
They all have different padding dimensions. Would it be best to have a common one for the font, background, margin, then add another class for the specific padding changes?